planting/planting_module/network/process.py

```diff
@@ -32,20 +32,20 @@ def register_machine(self, machine):
 
     def __call__(self, process, state):
         self.build_tasks(process, state)
-        self._planting.run_planting([self._env.ip], self._tasks)
-        self.print_info(self._planting)
-        return self._planting.result()
-
-    def print_info(self, planting):
+        return self.play()
+
+    def print_info(self):
+        res = self._planting.result()
+        state = self.active_state(self, self._planting)
+        if res is True:
+            self._planting.logger.info(
+                "host {}: ".format(self._env.ip) +
+                "handle service success, now active state is {0}"
+                .format(state))
+        else:
+            self._planting.logger.error("unarchive failed!")
```

Comment:
it's a mistake, and suggest failure message output more information may be like the success message